src/components/AccessibilityRequestsTable/index.tsx

```diff
@@ -64,13 +47,13 @@ const AccessibilityRequestsTable: FunctionComponent<AccessibilityRequestsTablePr
       },
       {
         Header: t('requestTable.header.businessOwner'),
-        accessor: (row: AccessibilityRequestsTableRow) => {
+        accessor: (row: AccessibilityRequests) => {
           return `${row.system.businessOwner.name}, ${row.system.businessOwner.component}`;
         }
       },
       {
         Header: t('requestTable.header.testDate'),
-        accessor: 'relevantTestDate',
+        accessor: 'relevantTestDate.date',
```

Comment:
What does the `accessor` do if the `relevantTestDate` is `null`, does it puke or handle it gracefully?

Reply:
Good question. This is under the hood of the table library we use, so I can't answer for sure without diving deeper. However, looking at this screenshot, it looks like it does just fine.

![Screen Shot 2021-02-24 at 10 16 11 AM](https://user-images.githubusercontent.com/8367504/109046204-54c92e80-7689-11eb-96c5-fe325eac8f3f.png)

Reply:
👍